sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -219,11 +219,15 @@ public static void validateValue(byte[] value) throws StateStoreException {
     public static boolean isSuppressed(StateStore stateStore) throws StateStoreException {
         byte[] bytes = fetchPropertyOrEmptyArray(stateStore, SUPPRESSED_PROPERTY_KEY);
 
-        try {
-            return new JsonSerializer().deserialize(bytes, Boolean.class);
-        } catch (IOException e){
-            LOGGER.error(String.format("Error converting property '%s' to boolean.", SUPPRESSED_PROPERTY_KEY), e);
-            throw new StateStoreException(Reason.SERIALIZATION_ERROR, e);
+        if (bytes.length == 0) {
```

Comment:
I can understand shortcutting for the trivial case (and being averse to exception-as-control-flow), but does it really buy us anything right now over the slightly more readable previous formulation?